.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com
+  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com,www.high-mobility.com
```

Comment:
@doomspork If I don't, the build fails with [this error](https://travis-ci.org/doomspork/elixir-companies/builds/268038755#L586):

```
 1. [L008]  https://www.high-mobility.com/ SSL_connect SYSCALL returned=5 errno=0 state=unknown state 
```